### PR TITLE
Added Offset option

### PR DESCRIPTION
**Pull request by dpaquette**
Added an Offset option:  An integer specifying the number of pixels to
offset when snapping to a panel.

This is useful when the page has a fixed position menu at the top.

### CODE REVIEW

**Comment by guidobouman** [comment]
Cool, great PR!

If you fix the whitespace issues I'll gladly merge this in.

**Comment by guidobouman** [comment]
This will go into v0.14.0. Sweet.

**Comment by dpaquette** [comment]
Great! I will fix up the whitespace issues this morning.

**Comment by dpaquette** [comment]
Okay, fixed the tabs and random whitespace

File: demos/index.html [hunk 1]
```diff
@@ -160,7 +169,7 @@
         right: 50px;
         left: 400px;
         overflow: scroll;
-      }
+      }
```
Comment (guidobouman) [inline demos/index.html]: So close!

**Comment by dpaquette** [comment]
This time for sure!

**Comment by guidobouman** [comment]
LGTM! :star: 

Merging when Travis tells me it's fine to do so.

**Comment by guidobouman** [comment]
Thanks for your work.

**Pull request merged**